doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -204,6 +204,8 @@ the system are restarted.
 | `updates_staged`       | [`bigint`]     | The number of updates (insertions plus deletions) the worker has written but not yet committed to the storage layer.                                                                                                                                                                |
 | `updates_committed`    | [`bigint`]     | The number of updates (insertions plus deletions) the worker has committed to the storage layer.                                                                                                                                                                                    |
 | `bytes_received`       | [`bigint`]     | The number of bytes the worker has read from the external system. Bytes are counted in a source type-specific manner and may or may not include protocol overhead.                                                                                                                  |
+| `envelope_state_bytes` | [`bigint`]     | The esimated number of bytes stored in the source envelope state. Note, these are currently rough estimates and may not reflect the exact value.                                                                                                                                    |
```

Comment:
```suggestion
| `envelope_state_bytes` | [`bigint`]     | The estimated number of bytes stored in the source envelope state. Note, these are currently rough estimates and may not reflect the exact value.                                                                                                                                    |
```

Reply:
Thanks for catching this 🙈

Reply:
Resolving since the word is removed in the next commit